composer.json

```diff
@@ -55,7 +55,7 @@
         "utopia-php/database": "0.59.0",
         "utopia-php/domains": "0.5.*",
         "utopia-php/dsn": "0.2.1",
-        "utopia-php/framework": "0.33.*",
+        "utopia-php/framework": "dev-fix-alias-params as 0.33.99",
```

Comment:
Let's add a todo to change it later

Reply:
Added linear task, framework needs extensive reviews so best not to be blocked by it's release